### ISSUE TITLE
Write IntrAdd to desc fields

### ISSUE THREAD

**Issue by BurTah00** [openscd/open-scd#362]
Hi,
This is the only not ported feature of Java Based version to the new one.
This is important for NA market especially for SEL and also SEL projects in Europe.
It is popping up so often nowadays.

I think you know the details.
Thanks!
Best Regards

**Comment by JakobVogelsang** [comment]
You are right this is botherin me as well. I see if I can squese that in.

**Comment by JakobVogelsang** [comment]
I have closed #127 as this issue is very close to this. This is the text within:

```
Hi,
You might recognize this feature that when @JakobVogelsang implemented it for SEL.
The same request popped up for ABB. See snipped from SCL below;


<Inputs>
<ExtRef iedName="REF620" ldInst="LD0" prefix="MV" lnClass="GAPC" lnInst="1" doName="Ind3" daName="stVal" intAddr="01-0C-CD-01-00-02,0002,1,
**GOOSERCV_BIN.3.I1,300,0,GOOSERCV_BIN**,Dynamic" serviceType="GOOSE" srcLDInst="LD0" srcCBName="gcbGooseBS_1" />
[Omicron_2.zip](https://github.com/openscd/open-scd/files/5988441/Omicron_2.zip)


<ExtRef iedName="REF620" ldInst="LD0" prefix="MV" lnClass="GAPC" lnInst="1" doName="Ind3" daName="q" serviceType="GOOSE" srcLDInst="LD0" srcCBName="gcbGooseBS_1" />

This feature will be very well reacted by ABB.

Thanks!
Burak
```
